doc/interop-test-descriptions.md

```diff
@@ -681,6 +695,14 @@ When the client requests COMPRESSABLE payload, the response includes a payload
 of the size requested containing all zeros and the payload type is
 COMPRESSABLE.
 
+### Echo Status
+[Echo Status]: #echo-status
+When the client sends a response_status in the request payload, the server returns
```

Comment:
Based on the text written, I would never expect the server to experience more than one message with response_status. Instead, I would expect the server to immediately close the stream when it gets the first response_status and so would ignore any further messages. Is that not the intention?